clusterman/batch/autoscaler_bootstrap.py

```diff
@@ -90,9 +90,13 @@ def configure_initial(self) -> None:
             self.options.pool,
             self.options.scheduler,
         )
-        self.config.watchers.append(
-            {self.options.pool: get_pool_config_path(self.options.cluster, self.options.pool, self.options.scheduler)},
-        )
+        watcher_config = {
+            self.options.pool:
+                get_pool_config_path(self.options.cluster,
+                                     self.options.pool,
+                                     self.options.scheduler)
+        }
+        self.add_watcher(watcher_config)
```

Comment:
nit: Perhaps comment here that the watchers don't reload anything. They watch configs and they notice that anything changes, yelp_batch's version checker will restart the entire batch.